pre-stylish.diff

```diff
@@ -0,0 +1,66 @@
+diff --git a/plutus-pab-client/pab-demo-scripts.nix b/plutus-pab-client/pab-demo-scripts.nix
```

Comment:
Delete (I know it's generated by stylish-haskell somehow, it happens on my machine too)

Reply:
Glad to :)